README.md

```diff
@@ -51,3 +60,24 @@ lrwxrwxrwx 1 root root 11 Oct  4 13:10 node_modules/goban -> ../../goban
 $ make
 ```
 
+
+# Running & Writing tests
+Is easy.
+
+## Deps
+Just install `jest` package (might need `sudo`):
```

Comment:
huh, I most definitely did not run `yarn install` myself, nor is it a step in makefile; so I am gonna add the `yarn install` step to README to the setup part, this will remove need for installing the `webpack` dep as well